tequilapi/endpoints/identities.go

```diff
@@ -51,9 +51,23 @@ type identityUnlockingDto struct {
 	Passphrase *string `json:"passphrase"`
 }
 
+// swagger:model PayoutInfoDTO
+type payoutInfoDto struct {
+	// in Ethereum address format
+	// required: true
+	// example: 0x000000000000000000000000000000000000000a
+	EthAddress *string `json:"eth_address"`
```

Comment:
if it's required, why is it a pointer?

Reply:
in such way it can be validated:
```
if req.EthAddress == nil {
	errors.ForField("eth_address").AddError("required", "Field is required")
}
```

Reply:
instead of pointer you can do req.EthAddress == "" and achieve the same result. Golang say - reuse zero values of types (zero value for string is "")

Reply:
updated, not using pointer here anymore